examples/vanillajs/main.js

```diff
@@ -0,0 +1,15 @@
+import { setupWalletSelector } from "@near-wallet-selector/core";
+import { setupModal } from "@near-wallet-selector/modal-ui-js";
+import { setupMyNearWallet } from "@near-wallet-selector/my-near-wallet";
+import { setupHereWallet } from "@near-wallet-selector/here-wallet";
+
+const selector = await setupWalletSelector({
+  network: "testnet",
+  modules: [setupMyNearWallet(), setupHereWallet()],
+});
+
+const modal = setupModal(selector, {
+  contractId: "test.testnet",
+});
+
```

Comment:
Could you add the `selector` and `modal` into the global `window` object?

```ts
window.selector = selector;
window.modal = modal;
```

Since the html document is pretty empty we can at least interact with the wallet selector through DevTools of the browser.